_dev/src/assets/json/googleUrl.json

```diff
@@ -36,7 +36,7 @@
   "googleAdsAccountCancellation" : "https://support.google.com/google-ads/answer/7619129",
   "googleAdsAccountSuspension" : "https://support.google.com/google-ads/answer/9841640",
   "learnAboutTimeZoneAndCurrency" : "https://support.google.com/google-ads/answer/9841530",
-  "googleAdsTermsAndCondition" : "https://support.google.com/adspolicy/answer/54818",
+  "googleAdsTermsAndCondition" : "https://support.google.com/adspolicy/answer/54818?hl=en",
```

Comment:
This is going to force the language in English for everyone, are we sure about this?

Reply:
problem is that in french we have an empty section

Reply:
Looks like the additional content is only for English speaking peaople, as this is about Australia.